DevCycleTests/Networking/DevCycleServiceTests.swift

```diff
@@ -72,11 +72,50 @@ class DevCycleServiceTests: XCTestCase {
     }
     
     func testProcessConfigReturnsNilIfBrokenJson() throws {
-        let service = getService()
         let data = "{\"config\":\"key}".data(using: .utf8)
         let config = processConfig(data)
         XCTAssertNil(config)
     }
+    
+    func testFlushingEvents() {
```

Comment:
Can you make sure there is a test that if there are less than 100 events it still gets eventually flushed.

Reply:
yup, this test is only generating 20 events :) the next one batches them with 205 events